alpha_challenges/the_basics.md

```diff
@@ -143,8 +149,9 @@ end
 **Answer 11:**
 ```ruby
 def names
-  # write your code in here
+  puts ["Jackie", "Hami"]
```

Comment:
The `puts` will print out the values for you, but it's not needed in order to return a value. So while it's great for debugging and making sure your code works, you don't need it to return the array 😀

Reply:
Oh, I see, so just running the code as:
def names
      ["Jackie", "Hami"]
end
would still print out Jackie and Hami?

Reply:
For some reason when I try running the code without the 'puts' it does not print out the value. 
def names
  ["Jackie", "Hami"]
end
return names
Do I have to change something else about the code after removing 'puts'?

Reply:
You would have to do: 
```
def names
  ["Jackie", "Hami"]
end

puts names
```